site/shared/src/main/scala/shared/User.scala

```diff
@@ -11,3 +11,13 @@ case class User(
   githubUrl:  String,
   email:      String
 )
+
+case class UserProgress(
+  id:          Long,
+  userId:      Long,
+  libraryName: String,
+  sectionName: String,
+  method:      String,
+  args:        Option[String],
```

Comment:
Have we decided that we are storing such detailed information already or what is the purpose of having all these fields?